src/ui/public/agg_table/agg_table.html

```diff
@@ -8,12 +8,21 @@
   totalFunc="totalFunc">
 
   <div class="kbnAggTable__controls">
-    <small>Export:</small>&nbsp;&nbsp;
+    <small
+      i18n-id="common.ui.aggTable.exportLabel"
+      i18n-default-message="Export:"
+    ></small>&nbsp;&nbsp;
     <a class="small" ng-click="aggTable.exportAsCsv(false)">
-      Raw <i aria-hidden="true" class="fa fa-download"></i>
+      <span
+        i18n-id="common.ui.aggTable.rawLabel"
+        i18n-default-message="Raw "
```

Comment:
We can remove a space after label and move `icon` tag to the next line, like here
```
<span
        i18n-id="common.ui.aggTable.rawLabel"
        i18n-default-message="Raw"
></span>
<i aria-hidden="true" class="fa fa-download"></i>
```